src/AssetType/AssetType.php

```diff
@@ -127,7 +127,7 @@ protected function getDynamicPath($path)
         $path = explode('/', $path);
         $path = array_filter($path);
 
-        if (reset($path) != $this->getDir()) {
+        if (reset($path) != $this->getDir() && !$this->isVueApp($path)) {
```

Comment:
Why can't Vue Apps enter this conditional?

Also I was slightly confused at 1st bc the angular vue app entered in here, so its confusing when saying and not a vue app. Wonder if we can come up with a better method name of more like why we can't / can enter the conditional vs saying not a vue app.